Testscripts/Linux/TIMESYNC-NTP.sh

```diff
@@ -167,6 +167,28 @@ elif is_suse ; then
         exit 0
     fi
 
+elif [[ $(detect_linux_distribution) == coreos ]]; then
+    # Refer to https://github.com/coreos/docs/blob/6ecaa639f0dd2098de037310d97c192fc49fce17/os/configuring-date-and-timezone.md#time-synchronization
```

Comment:
set link to refer to master branch:
https://github.com/coreos/docs/blob/master/os/configuring-date-and-timezone.md#time-synchronization